File: class02/java/AP0202-ParOuImpar.java
import java.util.Scanner;
public class ParImpar {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Número: ");
        int num = sc.nextInt();
        if (num % 2 == 0) System.out.println("Par");
        else System.out.println("Impar");
    }
}

File: class02/cpp/AP0202-ParOuImpar.cpp
#include <iostream>
int main() {
    int num;
    std::cout << "Numero: "; std::cin >> num;
    if (num % 2 == 0) std::cout << "Par" << std::endl;
    else std::cout << "Impar" << std::endl;
    return 0;
}
